arrows/qt/widgets/MetadataView.cxx

```diff
@@ -29,13 +29,14 @@ namespace kvr = kwiver::vital::range;
 namespace kwiver {
 namespace arrows {
 namespace qt {
+
 QTE_IMPLEMENT_D_FUNC( MetadataView )
 
-// /////////////////////////////////////////////////////////////////////////////
+// ////////////////////////////////////////////////////////////////////////////
```

Comment:
Ack, I think the added space here is not desired (uncrustify missing feature). Same for `//BEGIN` and `//END` actually (they still work as folding markers with the space, but they highlight differently... though arguably I should try to fix that on the editor side...).

Alternatively, we could use some other character for section divisions (maybe `#`?) which would make the interruption less jarring.